# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/RemoteContentController.h
# Commit: febdcf7986a3
# Full Hash: febdcf7986a37651188476cd8253d20e9cea443c
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r 00d1c650d88f -r febdcf7986a3 gfx/layers/ipc/RemoteContentController.h
--- a/gfx/layers/ipc/RemoteContentController.h	Wed Jun 22 19:26:58 2022 +0000
+++ b/gfx/layers/ipc/RemoteContentController.h	Wed Jun 22 19:40:17 2022 +0000
@@ -81,6 +81,9 @@
 
   void CancelAutoscroll(const ScrollableLayerGuid& aScrollId) override;
 
+  void NotifyScaleGestureComplete(const ScrollableLayerGuid& aGuid,
+                                  float aScale) override;
+
   void ActorDestroy(ActorDestroyReason aWhy) override;
 
   void Destroy() override;
@@ -106,6 +109,10 @@
 
   void CancelAutoscrollInProcess(const ScrollableLayerGuid& aScrollId);
   void CancelAutoscrollCrossProcess(const ScrollableLayerGuid& aScrollId);
+  void NotifyScaleGestureCompleteInProcess(const ScrollableLayerGuid& aGuid,
+                                           float aScale);
+  void NotifyScaleGestureCompleteCrossProcess(const ScrollableLayerGuid& aGuid,
+                                              float aScale);
 };
 
 }  // namespace layers
